LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Effect_Sustain_LW.uc

```diff
@@ -0,0 +1,48 @@
+class X2Effect_Sustain_LW extends X2Effect_Sustain;
+
+function bool PreDeathCheck(XComGameState NewGameState, XComGameState_Unit UnitState, XComGameState_Effect EffectState)
```

Comment:
Is this mostly a copy of the code from the super class function, just modified? If so, that's worth saying in a comment at the head of the function (i.e. above this line) along with what is changed from the base implementation. Just saying that it makes Sustain chance depend on overkill damage would be sufficient for that last part.